src/ILocaleInfo.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export default interface ILocaleInfo {
+  platform: 'windows' | 'macos';
+  regionalFormat: string;
+  date: {
+    shortDate: string;
+    longDate: string;
+    shortTime: string;
+    longTime: string;
+    calendar: string;
```

Comment:
I think calendar type can be changed to enum.
and `firstDayOfWeek` to numbers set 1..7

Reply:
Turns out these members are unused, so I will remove them and flatten the structure.